### PR TITLE
NL: Add script to generate a CSV with name+description

### PR DESCRIPTION
**Pull request by pradh**
This PR takes code originally from @jehangiramjad and makes some minor tweaks.

It copies a saved version of DDD titles in GCS.

### CODE REVIEW

**Review by jehangiramjad — APPROVED** [review]
Thanks! Only nitty comment is that the comments in the script refer to '14k' and '101k' SVs and that will be updated whenever the numbers differ from those. We might want to use other generic name but that's something we can do later

File: tools/nl/embeddings/prep/gen_svs.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  # Get all the existing SVs!
+  existing_dfs = []
+  for svf in sorted(glob.glob(FLAGS.existing_svs_filepattern)):
+    # Get the 1300 curated svs (Demo_US_SVs)
+    existing_dfs.append(pd.read_csv(svf).fillna(""))
+  existing_df = pd.concat(existing_dfs)
+
+  print(len(input_df))
+  print(len(ddd_df))
+  print(len(existing_df))
```
Comment (shifucun) [inline tools/nl/embeddings/prep/gen_svs.py:85]: remove or make this full print sentence?

File: tools/nl/embeddings/prep/run.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
```
Comment (shifucun) [inline tools/nl/embeddings/prep/run.sh:2]: 2023

**Review by shifucun — APPROVED** [review]

File: tools/nl/embeddings/prep/gen_svs.py (hunk 1 above)
Comment (pradh) [reply tools/nl/embeddings/prep/gen_svs.py]: Removed

**Comment by pradh** [comment]
> Thanks! Only nitty comment is that the comments in the script refer to '14k' and '101k' SVs and that will be updated whenever the numbers differ from those. We might want to use other generic name but that's something we can do later

Done.

**Pull request merged**